src/tools/jest/testUtils.tsx

```diff
@@ -0,0 +1,42 @@
+import {NavigationContainer} from '@react-navigation/native';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+
+import * as React from 'react';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import {render} from '@testing-library/react-native';
+import {QueryClient, QueryClientProvider} from '@tanstack/react-query';
+import {RecoilRoot} from 'recoil';
+
+export const testQueryClient = new QueryClient({
```

Comment:
You'll be reusing the same query client between tests. I suggest making it a function and calling it in wrapper, util, or test to create new instance for each test.

```suggestion
export const createQueryClient = (overrideOptions) => new QueryClient({
```

Reply:
Actually I need it only in one place for now. I removed it from not needed places. If I need to use it in multiple places I'll follow your suggestion.